client/components/commititems.jsx

```diff
@@ -1,13 +1,12 @@
 import React, {Component} from 'react';
-import CommitItem from './commititem';
+import CommitItem from './commitItem';
```

Comment:
I noticed the filename change as well and we've been working off the commititem filename for the last few pull requests. 

I think the best thing is to checkout a new branch based off your most recent changes, pull --rebase off the current dev branch, and mostly defer to the upstream changes while saving your work on the commit visualization, as we've structured the files differently than you have them set up for router.